apps/remix-ide/src/app/editor/contextualListener.js

```diff
@@ -123,14 +123,15 @@ class ContextualListener extends Plugin {
   }
 
   _highlightInternal (position, node) {
+    if (node.nodeType == 'Block') return
```

Comment:
Can you change this to strict equality check `===` not `==`

Reply:
:) of course, thanks. you are checking the standart manually :)